tests/kernel/init/src/main.c

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Please wrap this test into an existing test in the same area, it is too small and basic to be its own test (tests are expensive) unless more is coming in this area.